staging/src/k8s.io/apiserver/pkg/storage/testing/utils.go

```diff
@@ -115,6 +115,16 @@ func ExpectNoDiff(t *testing.T, msg string, expected, got interface{}) {
 	}
 }
 
+func ExpectNoDiffWithOne(t *testing.T, msg string, expectedList []interface{}, got interface{}) {
+	t.Helper()
+	for _, expected := range expectedList {
+		if reflect.DeepEqual(expected, got) {
+			return
+		}
+	}
+	t.Errorf("%s: differs from all items, first: %s", msg, cmp.Diff(expectedList[0], got))
```

Comment:
nit: when I moved the rest of the tests to use `cmp.Diff`, it was asked to check that `cmp.Diff()` actually produced a diff when `reflect.DeepEqual() == false`, and if not, to fall back to the previous impl, see `ExpectNoDiff` - probably want the same here

Reply:
changed - PTAL